Package.swift

```diff
@@ -33,21 +39,46 @@ let package = Package(
                 .product(name: "Spezi", package: "Spezi"),
                 .product(name: "NIO", package: "swift-nio"),
                 .product(name: "NIOFoundationCompat", package: "swift-nio"),
-                .product(name: "OrderedCollections", package: "swift-collections")
+                .product(name: "OrderedCollections", package: "swift-collections"),
+                // We have an issue in Xcode projects when importing XCTBluetooth in a test target that it fails
+                // to link with SpeziFoundation. lol. :)
+                .product(name: "SpeziFoundation", package: "SpeziFoundation")
```

Comment:
Would be good to create an issue that keeps track of this and e.g. checks if this is resolved in the next version of Swift/Xcode.